### DISCUSSION TITLE
PS subfolders scan with ".NAME"

### DISCUSSION THREAD

**Discussion by Makein** [Rparadise-Team/Koriki#11] [Q&A] [answered]
### What happened?

I hide multidisc games in folder with "." before name. 
And koriki show not only my ".m3u", but roms inside ".Folder" too.
On miyoo "." mean hide from menu, but koriki scan them.
How to hide, disable scan?

### Version

v1.0

**Comment by eduardofilo** [comment, marked as answer]
Any directory configured in the .ini files in .simplemenu/section_groups will be recursively scanned by SimpleMenu, including those starting with a dot. What you can do is to put that directory with multidisk games outside the normal path, for example in roms/PS-MULTI, and then adjust the paths inside the .m3u files. For example:

```
../PS-MULTI/Metal Gear Solid Disk 1.chd
../PS-MULTI/Metal Gear Solid Disk 2.chd
```
